fe/pom.xml

```diff
@@ -311,8 +315,14 @@ under the License.
                 <groupId>org.apache.hadoop</groupId>
                 <artifactId>hadoop-client</artifactId>
                 <version>${hadoop.version}</version>
-                <scope>compile</scope>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/org.apache.zookeeper/zookeeper -->
+            <dependency>
```

Comment:
Why adding zookeeper?

Reply:
It's a transitive dependency, due to the existence of CVE, so needs to upgrade